vividus-bdd-engine/src/main/java/org/vividus/bdd/resource/BddResourceLoader.java

```diff
@@ -33,6 +33,7 @@
 public class BddResourceLoader implements IBddResourceLoader
 {
     private static final String SEPARATOR = "/";
+    private static final String FILE_URL_PREFIX = "file:/";
```

Comment:
```suggestion
    private static final String FILE_URL_PREFIX = "file://";
```

Reply:
Do not work for users with Windows-machines:
```Caused by: java.net.UnknownHostException: C```
for file://C:/Users/../Temp/6939260240601242227.table

Reply:
https://datatracker.ietf.org/doc/html/rfc8089#appendix-A

Reply:
Got it, thanks. So we'll skip `minimal representation` of a local file and use only `traditional`?

Reply:
I prefer to use `traditional` approach since `minimal` one is not supported by all OS-s/tools